### ISSUE TITLE
GriefProtection and Chestshops error

### ISSUE THREAD

**Issue by RandyServer9807** [Erigitic/TotalEconomy#260]
Sponge Version: 7.1.0 BETA-3033
Forge Version:  
Total Economy Version: 1.8.0

### Description of issue
Server crashes when someone joins, console spews out errors as it's building the world. From what I can tell it's from a chest shop that was inside of a claim. I've attempted deleting the claim and that has done nothing. Deleting TotalEconomy fixes the issue.

Is there a specific location the chest shop data is stored so I may delete it?

### Console errors

Console Errors on Boot: https://pastebin.com/vFPaLh6b

Crashlog: https://pastebin.com/cr7Xuhqv
### Steps to reproduce
Make a chest shop in a claim, restart server.

EDIT: Added the versions since I noticed I forgot to add them.

**Comment by MarkL4YG** [comment]
The crashlog sounds interessting. I'll take a closer look in a few hours.

Do you know how reliable this is related to GriefProtection? (also is this the real plugin name or what plugin exactly do you use?)
Did you run into these issues only within claims or just any protected region?

**Comment by RandyServer9807** [comment]
It's this plugin: https://ore.spongepowered.org/blood/GriefPrevention
They were working fine together until a chestshop was made in an admin claim. I saw the console started just spitting out errors so I tried to restart it, and I could no longer join. This was the first chest shop that was made so I don't know if it's just them in general or if it was because it was in a claim.

I've stopped using Total Economy in the meantime, but I kept a backup of the files from when it was last on so I'll try disabling the chest shops now.
